internal/polkavm/host_call/accumulate_functions_test.go

```diff
@@ -909,19 +902,11 @@ func TestAccumulate(t *testing.T) {
 			for i, v := range tc.initialRegs {
 				initialRegs[i] = v
 			}
-			hostCall := func(hostCall uint32, gasCounter Gas, regs Registers, mem Memory, x AccumulateContextPair) (Gas, Registers, Memory, AccumulateContextPair, error) {
-				gasCounter, regs, mem, x, err = tc.fn(gasCounter, regs, mem, x, tc.timeslot)
-				require.ErrorIs(t, err, tc.err)
-				return gasCounter, regs, mem, x, nil
-			}
-			gasRemaining, regs, _, ctxPair, err := interpreter.InvokeHostCall(
```

Comment:
why you removed the InvokeHostCall from the host function tests?

Reply:
Two reasons, first of all, we no longer have the `Instruction` structure and we do not pass the parsed instructions to the VM, but instead we pass the bytecode as a series of bytes that are being traversed at the time of running the program. This means that now we need build the program in order to pass it to the VM, a functionality that we don't have. And second, it doesn't actually make sense to instantiate the interpreter for the unit tests, as by definition it is supposed to test only the afferent function in isolation